src/auth.rs

```diff
@@ -108,6 +111,9 @@ struct OAuthWire {
     access_token: String,
     token_type: String,
     scope: String,
+    expires_in: Option<usize>,
+    refresh_token: Option<String>,
+    refresh_token_expires_in: Option<usize>,
```

Comment:
The size of `expires_in` and `refresh_token_expires_in` are currently fixed at `28800` and `15811200`, respectively, so we could size down these fields to be `Option<u16>` and `Option<u32>`. I just copied the precedent for the `Claims` struct here